src/ecanvasser/util/ecanvasser.util.ts

```diff
@@ -0,0 +1,131 @@
+import { HttpService } from '@nestjs/axios'
+import { Logger } from '@nestjs/common'
+import { lastValueFrom } from 'rxjs'
+import { ApiResponse, PaginationParams } from '../ecanvasser.types'
+
+const DEFAULT_PAGE_SIZE = 1000
+
+export class EcanvasserApiUtil {
```

Comment:
Yeah.. this definitely needs to not just be a "util" class.. 

This needs to be a`EvanvasserService` within a separate `EcanvasserModule`